storyblocks/face_detection/model/code/inference.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Module: inference
+Created: 2020-08-19
+
+Description:
+
+    inference functions for pytorch serving container
+
+Usage:
+
+    >>> import inference
+    >>> model = inference.model_fn()
+
+"""
+import datetime
+import json
+import os
+import sys
+import tempfile
+
+import boto3
+import cv2
+
+HERE = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Does it make sense to wrap these lines that get executed in some sort of setup() function and call that?

Reply:
I haven't done that traditionally and tbh I don't actually know what the "pythonic" way is. good question.